google/resource_storage_bucket_object_test.go

```diff
@@ -44,6 +45,45 @@ func TestAccGoogleStorageObject_basic(t *testing.T) {
 	})
 }
 
+func TestAccGoogleStorageObject_update(t *testing.T) {
+	bucketName := testBucketName()
+
+	writeFile := func(name string, data []byte) string {
+		h := md5.New()
+		h.Write(data)
+		data_md5 := base64.StdEncoding.EncodeToString(h.Sum(nil))
+
+		ioutil.WriteFile(name, data, 0644)
+		return data_md5
+	}
+	data_md5 := writeFile(tf.Name(), []byte("data data data"))
+	updated_data_md5 := writeFile(tf.Name()+".update", []byte("datum"))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			if err != nil {
+				panic(err)
```

Comment:
I see other code does this, but I'm surprised by the panic (which would end all testing from running, right?)

How about changing it to a `t.Errorf`? There's no point halting everything as it'll just fail anyway.

Reply:
Seems reasonable. I just copied what was done in the other tests. Should we keep it isync with what the other tests are doing, only change this one or convert the other tests as well?